test/components/type-utils.test.ts

```diff
@@ -6,16 +6,17 @@ import { test, expect } from 'vitest';
 import { printFlags } from '../../lib/components/type-utils';
 
 test('serialises node flags', () => {
-  expect(
-    printFlags(
-      ts.NodeFlags.HasAsyncFunctions + ts.NodeFlags.HasImplicitReturn + ts.NodeFlags.ContainsThis,
-      ts.NodeFlags
-    )
-  ).toEqual('ContainsThis | HasImplicitReturn | ReachabilityCheckFlags | HasAsyncFunctions | ReachabilityAndEmitFlags');
```

Comment:
This assertion included their private members: 
https://github.com/microsoft/TypeScript/blob/e9bcbe6ef706e0b5a34678964988eb6a9cd86cc6/src/compiler/types.ts#L839-L846

Updated the test to use `toContain` to only check public enum members